### ISSUE TITLE
ERC20 Forwarder

### ISSUE THREAD

**Issue by affankhan43** [BitGo/eth-multisig-v2#7]
Contract missing the forwarder for ERC20 token how can i forward all ERC20 tokens to main contract address(parentAddress)

**Comment by barathcj** [comment]
You can flush tokens on the forwarder contract

**Comment by affankhan43** [comment]
yes i used the flush token function but how can it flush automatically on token deposit

**Comment by cernicc** [comment]
You can't do that. When someone transfers some tokens to the forwarder they actually call erc20 contract with transfer command. There is no ether on forwarder for forward of tokens. When you call flush you pay for that forward transfer on erc20 contract.

**Comment by trongcauhcmus** [comment]
> You can't do that. When someone transfers some tokens to the forwarder they actually call erc20 contract with transfer command. There is no ether on forwarder for forward of tokens. When you call flush you pay for that forward transfer on erc20 contract.

you must pay for the transaction fee to collect the token from forwarder.
There is another solution, if the issuer token use ERC223 instead of ERC20. there is a function name token fallback. they will trigger a function in receiver to forward token.
More detail: https://medium.com/coinmonks/the-erc-223-evolution-eec0ef23e5cc
